includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -603,16 +623,18 @@ protected function generate_image_tags( $tags, $attachment_id ) {
 	 */
 	public function setup_fields_sections() {
 		add_settings_section( $this->get_option_name(), $this->provider_service_name, '', $this->get_option_name() );
+		$settings = $this->get_default_settings();
```

Comment:
We can be more explicit here, `$default_settings` sounds better to me.

Reply:
Sounds good, I can implement this request.